### PR TITLE
[FIX] survey: fix archive ribbon on survey list

### PR DESCRIPTION
**Pull request by angv-odoo**
This commit fixes the archive ribbon position to the right top part of survey in list view.

Task-3259131

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by amdi-odoo** [comment]
Hello @angv-odoo :slightly_smiling_face: 
By playing a bit with the display on your runbot, I noticed that for smaller screen sizes the archive ribbon wasn't correctly displayed like it used to be in 15.2 :confused: 
Your runbot : https://tinyurl.com/25gto9vx
15.2 : https://tinyurl.com/25rt8bke

Sorry :confused: Could you check the display again ?
Thank you!

**Comment by amdi-odoo** [comment]
Hello @angv-odoo 
Thanks ! :slightly_smiling_face: 
I took a look but now the archived ribbon is kind of hanging (instead of clipping to the right) on middle screen sizes and on large screen sizes it overlaps the schedule button : https://tinyurl.com/2jzptvz6  
For reference, in survey, the ribbon is always sticking to the right and is a little bit smaller which might help seeing the schedule button (or if it still overlaps couldn't you set the schedule button a little bit more to the left ? I don't know if it is possible) : https://tinyurl.com/2momv3yb  

Could you look into it a bit more? :pray: 
Thank you!

**Comment by amdi-odoo** [comment]
Hello @angv-odoo :slightly_smiling_face: 
Looking great! We're close!
Just a detail, I still see a little black and blue square under the banner here :
image
It seems to be caused by a `::after` on the ribbon.

After that we should be good :slightly_smiling_face: 
Thanks for your work!

File: addons/survey/static/src/scss/survey_survey_views.scss [hunk 1]
```diff
@@ -79,28 +79,31 @@
...
+            width: 125px;
 
             // Mobile: is in a corner, takes more place
             @include media-breakpoint-down(md) {
                 height: 100px;
-                width: 100px;
+                width: 125px;
             }
-
+
```
Comment (tde-banana-odoo) [inline addons/survey/static/src/scss/survey_survey_views.scss:90]: :socks: extra spaces to remove ?

**Review by tde-banana-odoo — COMMENTED** [review]
Much review :disguised_face: 

Side note: could you expand a bit commit message, just telling what is the issue (aka it is not correctly contained in their line in the kanban-list-view ?)

File: addons/survey/static/src/scss/survey_survey_views.scss (hunk 1 above)
Comment (angv-odoo) [reply addons/survey/static/src/scss/survey_survey_views.scss]: Corrected and finally all the checks are also greened :partying_face:

Comment (angv-odoo) [reply addons/survey/static/src/scss/survey_survey_views.scss]: Also, Should I mention in commit that it should be only deployed from 16.0 to 16.2? (there is no issue in new design: 16.3 and up )

**Comment by tde-banana-odoo** [comment]
@robodoo r+

**Pull request closed without merging**